package.json

```diff
@@ -40,14 +40,15 @@
     "url-loader": "^1.1.2"
```

Comment:
Is this being used? Review carefully your `dependencies` and `devDependencies` and remove unused packages.

Reply:
I used this url-loader in webpack.config.js for parsing `assets/img/HCA-logo.jpg`